index.html

```diff
@@ -2,12 +2,16 @@
 <html lang="pt-BR">
 
 <head>
+  <title>Soberana</title>
   <link rel="preconnect" href="https://fonts.googleapis.com">
   <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
   <meta charset="UTF-8" />
   <meta http-equiv="X-UA-Compatible" content="IE=edge" />
   <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-  <title>Soberana</title>
+  <meta property="og:title" content="Soberana" />
+  <meta property="og:url" content="https://soberanatv.github.io" />
+  <meta property="og:image" content="/assets/img/logos-soberana/logo-com-texto.png" />
+  <meta property="og:description" content="A Soberana é um coletivo marxista-leninista que atua na disputa pela consciência política dos trabalhadores na internet." />
   <link rel="icon" href="/favicon.png">
   <link href="https://fonts.googleapis.com/css2?family=Ubuntu&display=swap" rel="stylesheet">

```

Comment:
```suggestion
<link rel="preload" href="https://fonts.googleapis.com/css2?family=Ubuntu&display=swap" as="font">

```
Permite o texto ja ser lido enquanto a fonte vai sendo baixada depois

Reply:
também melhora a performance e o SEO